tests/estimators/classification/test_deeplearning_common.py

```diff
@@ -509,3 +508,90 @@ def test_learning_phase(art_warning, image_dl_estimator):
         assert hasattr(classifier, "_learning_phase")
     except ARTTestException as e:
         art_warning(e)
+
+
+@pytest.mark.skipModule("apex.amp")
```

Comment:
Hi Minh, I'm just realising now this minor issue. The previous message might have been confusing. Just to be clear, `def test_loss_gradient_amp` should be placed in `test_deeplearning_specific.py` since it is specific to pytorch. Apart from that all the tests look good ! Thanks for upgrading them to the new system :-)

Reply:
@killianlevacher : Done.